docs/en/concepts-and-designs/lal.md

```diff
@@ -0,0 +1,216 @@
+# Log Analysis Language
+
+Log Analysis Language (LAL) in SkyWalking is essentially a Domain-Specific Language (DSL) to analyze logs. You can use
+LAL to parse, extract, filter, analyze, and save the logs, as well as collaborate the logs with traces, and metrics.
+
+Check the [`default.yaml`](../../../oap-server/server-bootstrap/src/main/resources/lal/default.yaml) for examples.
+
+## Filter
+
+A filter is a group of [parser](#parser), [extractor](#extractor) and [sink](#sink). Users can use one or more filters
+to organize their processing logics. Every piece of log will be sent to all filters in an LAL rule. The piece of log sent into the filter is available as property `log` in the LAL, therefore you can access the log service name via `log.service`.
+
+### Parser
```

Comment:
How to merge multiple lines into a single one, for example, the error stack scenario?

Reply:
I think that is log collector or receiver responsibility, rather than LAL. Logic line identify is required before building Log structure.